### ISSUE TITLE
Only ADD/COPY `--from`  images in **trustedRegistries**

### ISSUE THREAD

**Issue by r10r** [hadolint/hadolint#1108]
Hi there,

we have a lot of Dockerfiles here that use `COPY --from` and `ADD --from`.
I want to use `hadolint` to verify that only images with domains listed in the **trustedRegistries** list are allowed.

How can that be accomplished ?

Thanks for your help!
r10r

**Comment by sakshamgoyal-01** [comment]
any progress on this?

**Comment by r10r** [comment]
> any progress on this?

It is targeted to milestone v2.15.0 ...
@m-ildefons Thanks a lot for implementing this 👍  Will there be a pre-release / release candidate for testing this before v2.15.0 ?

Alternatively [building from source](https://github.com/hadolint/hadolint#install) is possible if the feature is required ASAP.

**Comment by sakshamgoyal-01** [comment]
@r10r will it cover cases of this?
```dockerfile
RUN --mount=...,from=<image>
```

**Comment by m-ildefons** [comment]
Good point @sakshamgoyal-01 . At the moment DL3026 will not catch that.
But I've opened a PR to fix it here: https://github.com/hadolint/hadolint/pull/1216
Let me know if this covers your use case adequately.

**Comment by sakshamgoyal-01** [comment]
It's not my use case. I was just looking to see if there were any other places were an image is accepted but not checked with DL3026

**Comment by m-ildefons** [comment]
Fixed in v2.15.0
